src/v2/cookbook/form-validation.md

```diff
@@ -428,7 +428,7 @@ Comenzamos con una variable que respresenta la URL de la API que está ejecutand
 
 ## Patrones Alternativos
 
-Mientras este libro de recetas se basa en realizar la validación de formularios "a mano", existen, por supuesto, algunas librerías Vue que manejarán muchos de estos detalles por usted. Cambiar a una librería existente puede impactar en el tamaño final de su aplicación, pero los beneficios pueden ser tremendos. Tendrá código (probablemente) testado y también regularmente actualizado. Algunos ejemplos de librerías de validación de formularios para Vue incluyen:
+Mientras este libro de recetas se basa en realizar la validación de formularios "a mano", existen, por supuesto, algunas librerías Vue que manejarán muchos de estos detalles por usted. Cambiar a una librería existente puede impactar en el tamaño final de su aplicación, pero los beneficios pueden ser tremendos. Tendrá código probado (probablemente) y también regularmente actualizado. Algunos ejemplos de librerías de validación de formularios para Vue incluyen:

```

Comment:
Hey @aderegil, thanks for the contribution. I was discussing this change with the team. `Testear` is accepted by RAE, you can check it [here](https://dle.rae.es/testear).

Anyway, we need to fix the typo. Please, commit my following suggestion to approve the fix and merge.

```suggestion
Mientras este libro de recetas se basa en realizar la validación de formularios "a mano", existen, por supuesto, algunas librerías Vue que manejarán muchos de estos detalles por usted. Cambiar a una librería existente puede impactar en el tamaño final de su aplicación, pero los beneficios pueden ser tremendos. Tendrá código testeado (probablemente) y también regularmente actualizado. Algunos ejemplos de librerías de validación de formularios para Vue incluyen:

```